plutus-wallet-api/ledger/Ledger/Index.hs

```diff
@@ -270,15 +273,22 @@ checkPositiveValues t =
     then pure ()
     else throwError $ NegativeValue t
 
+-- | Check if the fees are paid exclusively in Ada.
+checkFeeIsAda :: ValidationMonad m => Tx -> m ()
+checkFeeIsAda t =
+    if (Ada.toValue $ Ada.fromValue $ txFee t) == txFee t
+    then pure ()
+    else throwError $ NegativeValue t
```

Comment:
Why not `FeeNotInAda` or something? `NegativeValue` doesn't really apply here (the fee should still be positive but that's not what `checkFeeIsAda` is checking)

Reply:
Oops thanks, copypaste error.